csrc/device_lower/pass/alias_memory.cpp

```diff
@@ -385,7 +385,7 @@ class BufferLiveInterval {
 
   std::string toString() {
     std::stringstream ss;
-    ss << "[ " << first_write_pos_ << " , " << last_read_pos_ << " )";
+    ss << "[ " << first_write_pos_ << " , " << last_read_pos_ << " ]";
```

Comment:
Print as closed intervals, which they always have been for inner liveness intervals, and now are also for outer live intervals.